sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -16153,6 +16161,44 @@ public void testTimeStampAddConversion() throws Exception
     );
   }
 
+  public void testSizeFormatFunction() throws Exception
+  {
+    testQuery(
+        "SELECT m1, "
+        + "BINARY_BYTE_FORMAT(45678),"
+        + "BINARY_BYTE_FORMAT(m1*12345),"
+        + "BINARY_BYTE_FORMAT(m1*12345, 0), "
+        + "DECIMAL_BYTE_FORMAT(m1*12345), "
+        + "DECIMAL_FORMAT(m1*12345) "
+        + "FROM numfoo WHERE f1 = 0.1 LIMIT 1",
+        ImmutableList.of(
+            newScanQueryBuilder()
+                .dataSource(CalciteTests.DATASOURCE3)
+                .intervals(querySegmentSpec(Filtration.eternity()))
+                .virtualColumns(expressionVirtualColumn("v0", "'44.61KiB'", ValueType.STRING),
```

Comment:
This is a special case. I also thought it should a function call expression here. But because the given argument is a constant, this function call has been calculated during SQL parsing phase before converting to native sql. I've added some comments in the latest commit here.

Reply:
👍  Thanks for clarifying